modules/Bio/EnsEMBL/Variation/DBSQL/VCFCollectionAdaptor.pm

```diff
@@ -208,6 +210,17 @@ sub new {
       }
     }
 
+    # Check that the remote/local VCF file exists before creating a corresponding VCFCollection object
+    my $filename_template = $hash->{filename_template} =~ /(nfs|ftp:)/ ? $hash->{filename_template} : $root_dir.$hash->{filename_template};
+
+    if ($filename_template !~ /[#]+[^#]+[#]+/) {
```

Comment:
Sometimes the filename is using # characters (e.g. for 1KG: ALL.chr###CHR###), so we can't really test them.
It's a quick fix for e93 but it might need deeper changes (ensembl-io) in the future